From 1735a12abc80ac9b975b8c46f1ff297704a62984 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Tue, 15 Oct 2024 07:43:51 +0200
Subject: [PATCH] Fix installation

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 90193f03..9400fa4e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -57,7 +57,7 @@ Package = "https://pypi.org/project/custodian"
 
 [tool.setuptools.packages.find]
 where = ["src"]
-include = ["custodian"]
+include = ["custodian*"]
 
 [tool.ruff]
 target-version = "py311"
